### PR TITLE
Updates internal github actions to the latest versions

### PR DESCRIPTION
**Pull request by martrapp**
#### Description

Get rid of those warnings:
image

### CODE REVIEW

**Review by kevinzunigacuellar — APPROVED** [review]
LGTM! Logs are looking nice (no warnings) and majors versions seem to have no breaking changes besides upgrading node 16 to 20.

File: .changeset/fluffy-trains-rest.md
Comment (delucis) [inline .changeset/fluffy-trains-rest.md:1]: No need for a changeset in this case — none of these impact integration users — so we can remove this.

**Comment by martrapp** [comment]
I kept actions/labeler@v4 as updating to v5 requires a change of the configuration. This will be a PR on its own.

**Review by delucis — CHANGES_REQUESTED** [review]
Thanks so much for the careful updates @martrapp!

Just one note from me.

**Review by delucis — APPROVED** [review]
Thanks again @martrapp — LGTM 🌟

**Pull request merged**